### PR TITLE
:arrow_up: Bump python to 3.8, mapentity to 8.3.0

### PR DESCRIPTION
**Pull request by Chatewgne**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -7,9 +7,9 @@ Build-Depends:
  debhelper (>= 9),
  dh-virtualenv (>= 1.0),
  git,
- python3 (>= 3.6),
- python3-venv (>= 3.6),
- python3-dev (>= 3.6),
+ python3.8 (>= 3.8),
+ python3.8-venv (>= 3.8),
+ python3.8-dev (>= 3.8),
```
Comment (marcantoinedupre) [inline debian/control:12]: You have added `sofware-properties-common` in the build dependencies of the CI's job but not here. Is that intended?

**Review by numahell — COMMENTED** [review]
Missing changelog maybe

**Comment by submarcos** [comment]
Add a warning to Bionic users to use external [PPA](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa) to keep python3.8 up to date

File: docs/changelog.rst [hunk 2]
```diff
@@ -5,6 +5,11 @@ CHANGELOG
 2.94.0+dev (XXXX-XX-XX)
 -----------------------
 
+**Maintenance**
+
+- Bump Python to 3.8
+- Bump MapEntity to 8.3.0
+- Add libvips to dependencies
```
Comment (submarcos) [inline docs/changelog.rst:12]: related to HDViewpoint PR

**Review by submarcos — COMMENTED** [review]
may be some changes are not used anymore

File: debian/control (hunk 1 above)
Comment (Chatewgne) [reply debian/control:12]: It was not used anymore so I removed it, thanks :)

**Pull request merged**